lib/Helpers/ASM.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * This helper builds the request body for a /mail/send API call.
+ *
+ * PHP version 5.6, 7
+ *
+ * @author    Elmer Thomas <dx@sendgrid.com>
+ * @copyright 2017 SendGrid
+ * @license   https://opensource.org/licenses/MIT The MIT License
+ * @version   GIT: <git_id>
+ * @link      http://packagist.org/packages/sendgrid/sendgrid
+ */
+
+namespace Helpers;
```

Comment:
The namespace here would be Mail and the directory would be Helpers/Mail. Perhaps we should move these classes to Helpers/Mail/Model, like [this](https://github.com/sendgrid/sendgrid-csharp/tree/master/src/SendGrid/Helpers/Mail/Model).